.changeset/major-remove-legacy-assets.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": major
+---
+
+Remove legacy assets
```

Comment:
I'm still seeing some legacy assets related code left in the code base...if you search for `legacyAssets`...

Reply:
I've renamed to clarify—most of the remaining code was Sites related, which we're not removing